sdk/src/kalix.ts

```diff
@@ -718,7 +718,8 @@ export class Kalix {
           proxyInfo.internalProxyHostname ?? proxyInfo.proxyHostname,
         proxyPort:
           this.proxyPort ??
-          (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80),
+          (proxyInfo.proxyPort ||
+            (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80)),
```

Comment:
what does this mean? if the `proxyInfo.proxyPort` is defined it will use port 9000? shouldn't it use the value from the `proxyInfo.proxyPort`?

Reply:
Some duplication here, `variable ?? elsevalue` will use `variable` if truthy and `elsevalue` if `variable` is falsy (undefined, null, empty string etc), so the `proxyPort || elseExpression` just duplicates that. 

```this.proxyPort ?? (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80)```

Should be enough.

Reply:
Ah, sorry, `??` is "nullish coalescing" so not quite the regular JS falsyness monkey business but an additional concept returning the value if it is for example `""` or `0` but never if `undefined` or `null`. Given that I think it should rather be only the right hand part to not be unnecessary hard to read: 

```
  (proxyInfo.proxyPort || (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80))
```

Reply:
before there was `this.proxyPort ||
            (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80)`

Reply:
Ah, I missed `this` in there, so that's when it comes from the integration testkit I guess. 👍 NVM my noise then.

Reply:
:) that's not noise. Thanks for the input, Johan. Yes, that's used by the integration test kit.

Reply:
what is weird is that I would swear I added a comment to your comment pointing exactly that. I may have missed hitting `comment` I guess.